grammars/prql-lezer/test/operators.txt

```diff
@@ -62,10 +62,18 @@ filter foo ?? bar
 
 Query(Pipeline(CallExpression(Identifier,ArgList(BinaryExpression(Identifier,LogicOp,Identifier)))))
 
-# Unary operator
+# Unary operator -
 
 sort { -name }
 
 ==>
 
 Query(Pipeline(CallExpression(Identifier,ArgList(TupleExpression(UnaryExpression(ArithOp,Identifier))))))
+
+# Unary operator ==
+
+filter ==customer_id
```

Comment:
Yes, I know, but thanks for pointing it out which helped me get back on track. The reason I left the parentheses out was because the parser didn't support those at the time, but now with PR #3906 it does, and which is why this PR was a draft, because things weren't ironed out yet.